reports/Tenant Weekly Alert Report.json

```diff
@@ -2,20 +2,14 @@
     "@context": "\/api\/3\/contexts\/Reporting",
     "@type": "Reporting",
     "uuid": "b4a56121-a556-4932-a573-13807d0c678d",
-    "createUser": "\/api\/3\/people\/3451141c-bac6-467c-8d72-85e0fab569ce",
-    "createDate": 1645545844.548621,
-    "modifyUser": "\/api\/3\/people\/3451141c-bac6-467c-8d72-85e0fab569ce",
-    "modifyDate": 1645545971.750825,
-    "recordTags": [],
     "displayName": "Tenant Weekly Alert Report",
     "type": "rows",
     "templateType": "reporting",
     "roles": [
         "\/api\/3\/roles\/17562d60-b841-4ed7-912b-1ebaf063c8c9",
         "\/api\/3\/roles\/21d7a7c6-9d68-438f-b732-e8950c5f745b",
         "\/api\/3\/roles\/a501c11e-15af-49a6-a3a4-88605f32f016",
-        "\/api\/3\/roles\/c5ff8174-b3fa-41c5-b781-2b890b426a80",
-        "\/api\/3\/roles\/aada6692-fbf4-4a02-a580-f8a200962365"
```

Comment:
which roles were removed and added?

Reply:
No new Role is added only the T2 analyst Role is converted to SOC Analyst and added that in Report.